management-interface.go

```diff
@@ -576,18 +614,46 @@ func getInstalledPackages() (string, error) {
 }
 
 // AboutHandler shows the currently installed packages on the device.
-func AboutHandler(w http.ResponseWriter, r *http.Request) {
+func AboutHandler(w http.ResponseWriter, r *http.Request, apiObj *api.ManagementAPI) {
```

Comment:
apiObj isn't being used. 

The rest of this function looks great.

Reply:
Sorry, meant to remove that. Done now.